scripts/prepare.py

```diff
@@ -6,8 +6,17 @@
 
 from titlecase import titlecase
 
-def process(extracted_files):
+data_file_path = os.path.join('data', 'subdivision-codes.csv')
+
+def remove_double_quotes(file_path):

```

Comment:
This is removing all double quotes? What about when it is necessary? (example when there's a comma in the field

Reply:
@sabas I have looked into old data there is an example like this https://github.com/datasets/un-locode/blob/6e87b5b4a0bd577a2a972926649a9ce321abbf09/data/country-codes.csv#L28 which doesn't have double quotes, that's why I thought it is unnecessary, should I change it with csv handler in this case?

Reply:
If you remove the double quotes from BO,"Bolivia, Plurinational State of" 
the csv then has three fields instead of two, so it is necessary to make Bolivia, Plurinational State of appear as one field

Reply:
@sabas Ok I will replace with a csv handler then!

Reply:
@sabas  I have tested csv handler, since my logic is based on processing using `scripts/prepare.py` I have just adjusted `remove_double_quotes()` function with `QUOTE_MINIMAL` which I think done a great job leaving only necessary double quotes wdyt?